client/blocks/upwork-banner/style.scss

```diff
@@ -0,0 +1,79 @@
+.upwork-banner {
+	background: no-repeat var( --color-accent );
+	display: block;
+	overflow: hidden;
+	padding: 0.8em;
+	position: relative;
+	user-select: none;
+
+	h1,
+	p,
+	.button.upwork-banner__cta {
+		color: var( --color-neutral-700 );
+		position: relative;
+		z-index: z-index( '.upwork-banner', '.button.upwork-banner__cta' );
+	}
+
+	h1 {
+		font-size: 1.6em;
+		font-weight: 500;
+		letter-spacing: 0.01em;
+		margin: -0.1em 0;
+		line-height: 1.3;
+
+		+ p {
+			margin-top: 0.6em;
+		}
+	}
+
+	p {
+		margin: 0;
+	}
+
+	img {
+		transform-origin: top right;
+		max-width: 190px;
+		transition: opacity 400ms ease-in-out;
+		position: absolute;
+		top: 0;
```

Comment:
These position values need to be indented too.

Reply:
done